.github/pull-request-template.md

```diff
@@ -0,0 +1,34 @@
+<!-- Please follow the guidelines at https://www.conventionalcommits.org/en/v1.0.0/ and use one of the following in your title:
+feat:            <-- New features that require a MINOR version update
+fix:             <-- Bug fixes that require at PATCH version update
+docs:            <-- Documentation change that does not impact code
+chore:           <-- Metadata changes such as dependency update or configuration files
+test:            <-- Test changes that do not impact behavior
+perf:            <-- Code changes that improve performance but do not impact behavior
+BREAKING CHANGE: <-- Include if your change includes a backwards incompatible change
+-->
+
+Fixes # <!-- issue number -->
+
+**Description**
+
+**How was this change tested?**
+
+*
```

Comment:
Extra '*'?

Reply:
This is just a prompt to use bullet points. I can remove though, wdyt?

Reply:
Weird, I didn't know that did bullets. I always did dashes, LGTM!